Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -458,16 +468,34 @@ export default class TimelineView extends Vue {
                     </b-row>
                 </div>
                 <LinearTimeline
-                    v-if="isLinearView && !isLoading"
+                    v-show="isLinearView && !isLoading"
                     :timeline-entries="filteredTimelineEntries"
                     :total-entries="getTotalCount()"
                 >
+                    <template #add-note>
+                        <b-col
```

Comment:
Note: columns are embedded inside rows otherwise there are alignment issues. Since the slot is passing a column the internal positioning might cause problems if the slot in the child component is not wrapped in a row. To avoid this we could wrap the slot placeholder as a column and have the parent only pass the component that it needs without worrying about its positioning.
Another problem is that by having the col on the parent the positioning of the slot _leaking_ implementation details

Parent:
```
<template #add-note v-if="isNoteEnabled && isLinearView && !isLoading">
   <add-note-button>
   ...
   <add-note-button />
</template >
```

Child
```
         <b-row>
            <b-col>
                <CalendarHeader
                    class="sticky-top sticky-offset p-2"
                    :class="{ 'header-offset': isHeaderShown }"
                    :current-month.sync="currentMonth"
                    :title-format="titleFormat"
                    :available-months="availableMonths"
                />
            </b-col>
            <b-col v-if="isNoteEnabled" col cols="auto">
                <add-note-button text-breakpoint="md" class="p-2" />
            <b-col> <!--- Up to the parent to position it properly --->
                 <slot name="add-note" />
             </b-col>
        </b-row>
```

Reply:
I was trying to avoid having to define isNoteEnabled in the child components, but I can make the change if you think it's cleaner overall.

Reply:
I think is cleaner, but We can change it later.